src/main/java/org/isf/menu/rest/UserController.java

```diff
@@ -65,29 +63,24 @@
 
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
```

Comment:
Please follow project standards for ordering of imports;; this import was in the correct location before your change; thanks